web/template/home.gohtml

```diff
@@ -365,7 +365,7 @@
                                                 <template x-if="livestream.InLectureHall()">
                                                     <a class="tum-live-badge bg-black"
                                                        target="_blank"
-                                                       :href="livestream.LectureHall.ExternalURL">
+                                                       :href="livestream.LectureHall.ExternalURL">s.
```

Comment:
What  is this "s." at the end of the line for?

Reply:
Oops, it seems i overlooked it. I will correct this asap.